### DISCUSSION TITLE
Preview height when vertical layout

### DISCUSSION THREAD

**Discussion by idr4n** [ibhagwan/fzf-lua#1364] [Q&A] [answered]
Hi,

I was wondering if it is possible to achieve something similar to the preview in Telescope's `get_dropdown` theme. That is, the preview is on top of the picker window and occupies whatever space is available (whiteout shrinking the picker window). In Fzflua I get a similar look, but when I trigger the preview, the picker window adjusts accordingly.

Here is a video for clarification:

https://github.com/user-attachments/assets/69dc0ada-66c7-47dc-a3be-db7255d81366

My current settings are these one:

```lua
  opts = function()
    return {
      winopts = {
        preview = {
          vertical = "up:50%",
          layout = "vertical",
          scrollbar = false,
        },
      },
      winopts_fn = function()
        local max_width = 120 / vim.o.columns
        local max_height = 20 / vim.o.lines
        return {
          backdrop = 100,
          width = math.min(max_width, 0.8),
          height = math.min(max_height, 0.8),
        }
      end,
      fzf_opts = {
        -- ["--layout"] = "default",
        ["--layout"] = "reverse",
        ["--info"] = "inline-right",
        -- ["--layout"] = "reverse-list",
      },
}
```

Thanks in advance!

**Comment by ibhagwan** [comment, marked as answer]
This is possible with `previewers.builtin.toggle_behavior = 'extend'`, see #721 for more info and how to configure:
```lua
require("fzf-lua").setup({
  previewers = { builtin = { toggle_behavior = "extend" } },
})
```

**Reply by idr4n** [reply]
Thanks a lot for the quick response. That's what I was looking for. Sorry I missed 721.
